### PR TITLE
upgrade elliptic version to address vulnerability

### PR DESCRIPTION
**Pull request by dasormeter**
Upgrade elliptic version
image

### CODE REVIEW

File: docs/theme/package.json [hunk 1]
```diff
@@ -3,6 +3,9 @@
   "main": "js/theme.js",
   "version": "1.0.0",
   "private": true,
+  "resolutions" : {
+    "**/elliptic": "6.5.3"
```
Comment (cocreature) [inline docs/theme/package.json:7]:
```suggestion
    "**/elliptic": "^6.5.3"
```

File: navigator/frontend/package.json [hunk 2]
```diff
@@ -85,6 +85,7 @@
   },
   "resolutions": {
     "**/@types/react": "~16.0.0",
-    "**/apollo-client": "~1.4.0"
+    "**/apollo-client": "~1.4.0",
+    "**/elliptic": "6.5.3"
```
Comment (cocreature) [inline navigator/frontend/package.json:89]:
```suggestion
    "**/elliptic": "^6.5.3"
```

File: package.json [hunk 3]
```diff
@@ -4,6 +4,9 @@
   "description": "package for bazel-managed nodejs dependencies",
   "private": true,
   "version": "0.0.0",
+  "resolutions" : {
+    "**/elliptic": "6.5.3"
```
Comment (cocreature) [inline package.json:8]:
```suggestion
    "**/elliptic": "^6.5.3"
```

File: docs/theme/package.json (hunk 1 above)
Comment (hurryabit) [reply docs/theme/package.json:7]: Doesn't `^6.5.3` mean semver-compatible with `6.5.3`? If so, that would include `6.5.2` as well. We probably need something like
```suggestion
    "**/elliptic": "^6.5.3 >=6.5.3"
```
Does that look right?

Comment (cocreature) [reply docs/theme/package.json:7]: I don’t think so, `^` also adds a lower bound. See https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004

Comment (hurryabit) [reply docs/theme/package.json:7]: Yeah, you're right. I stopped reading too early. Apologies.

Comment (dasormeter) [reply docs/theme/package.json:7]: @cocreature -- added the resolutions entry on package.json then reran yarn install so no manual edit of lock file

Comment (dasormeter) [reply docs/theme/package.json:7]: i'll add the range rather than specific version to both

File: navigator/frontend/package.json (hunk 2 above)
Comment (dasormeter) [reply navigator/frontend/package.json]: addressed

**Pull request merged**